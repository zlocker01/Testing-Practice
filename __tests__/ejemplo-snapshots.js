const cliente = {
    nombre: 'juan 2',
    balance: 500,
    tipo: 'premium'
};

describe('Test al cliente', () => {
    test('Es juan', () => {
        expect(cliente).toMatchSnapshot();
    });
});