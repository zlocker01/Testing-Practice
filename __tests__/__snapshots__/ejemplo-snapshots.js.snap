// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test al cliente Es juan 1`] = `
{
  "balance": 500,
  "nombre": "juan 2",
  "tipo": "premium",
}
`;
